docs/Set-PSReadLineOption.md

```diff
@@ -500,7 +500,7 @@ Aliases:
 
 Required: False
 Position: Named
-Default value: On Windows - ~\AppData\Roaming\Microsoft\Windows\PowerShell\PSReadLine\$($host.Name)_history.txt otherwise $XDG_DATA_HOME/powershell/PSReadLine/$($host.Name)_history.txt or $HOME/.local/share//powershell/PSReadLine/$($host.Name)_history.txt
+Default value: A file named $($host.Name)_history.txt in $env:APPDATA\Microsoft\Windows\PowerShell\PSReadLine on Windows, $env:XDG_DATA_HOME/powershell/PSReadLine on macOS, and $env:HOME/.local/share/powershell/PSReadLine on Linux
```

Comment:
The code literally checks XDG_DATA_HOME and HOME on non-Windows platforms, and I think the code is correct in that XDG_DATA_HOME is used on some Linux variants and HOME is set on macOS, so either variable should work on both macOS and Linux.

Reply:
Is this OK then:
A file named $($host.Name)_history.txt in $env:APPDATA\Microsoft\Windows\PowerShell\PSReadLine on Windows and $env:XDG_DATA_HOME/powershell/PSReadLine, or $env:HOME/.local/share/powershell/PSReadLine on non-Windows platforms

Reply:
Yes.